.github/workflows/release.yaml

```diff
@@ -74,3 +109,94 @@ jobs:
         uses: github/codeql-action/upload-sarif@v2
         with:
           sarif_file: snyk.sarif
+
+  binary-provenance:
+    needs: [ goreleaser ]
+    permissions:
+      actions: read # To read the workflow path.
+      id-token: write # To sign the provenance.
+      contents: write # To add assets to a release.
+
+    # Note: this _must_ be referenced by tag. See: https://github.com/slsa-framework/slsa-verifier/issues/12
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@v1.9.0
+
+    with:
+      base64-subjects: "${{ needs.goreleaser.outputs.hashes }}"
```

Comment:
Could you please explicitly specify the `provenance-name: "openfga.intoto.jsonl"`